pkg/quota_management/billing_model.go

```diff
@@ -1,10 +1,20 @@
 package quota_management
 
+import "time"
+
 type BillingModel struct {
-	Id                  string `yaml:"id"`
-	ExpirationDays      int    `yaml:"expiration_days"`
-	GracePeriodDays     int    `yaml:"grace_period_days"`
-	MaxAllowedInstances int    `yaml:"max_allowed_instances"`
+	Id                  string          `yaml:"id"`
+	ExpirationDate      *ExpirationDate `yaml:"expiration_date,omitempty"`
+	GracePeriodDays     int             `yaml:"grace_period_days"`
+	MaxAllowedInstances int             `yaml:"max_allowed_instances"`
+}
+
+func (bm *BillingModel) IsExpired() bool {
+	if bm.ExpirationDate == nil {
+		return false
+	}
+	now := time.Now()
+	return now.After(time.Time(*bm.ExpirationDate))
```

Comment:
what about having a `HasExpired` method in ExpirationDate?

Reply:
That's a good idea.